gulp-tasks/publish-glitch.js

```diff
@@ -0,0 +1,58 @@
+/*
+  Copyright 2018 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+const gulp = require('gulp');
+const childProcess = require('child_process');
+const fse = require('fs-extra');
+const tempy = require('tempy');
+const util = require('util');
+const del = require('del');
+
+const exec = util.promisify(childProcess.exec);
+
+const key = '9c5af840-16dd-42d8-b5d1-0fa4a52939b8';
+
+const glitchProjects = [
+  'workbox-core',
+  // 'workbox-sw',
```

Comment:
FYI, you still have these commented out.

Reply:
yeah still need to update the server in each glitch project but can finish by end of this week